src/containers/AccountConnection.jsx

```diff
@@ -385,8 +384,7 @@ const mapStateToProps = (state, ownProps) => ({
   success: isConnectionConnected(state.connections, ownProps.trigger),
   deleting: isConnectionDeleting(state.connections, ownProps.trigger),
   error: getConnectionError(state.connections, ownProps.trigger),
-  queued: isConnectionEnqueued(state.connections, ownProps.trigger),
-  accountsCount: getKonnectorTriggersCount(state, ownProps.konnector)
```

Comment:
@gregorylegarec do you confirm that we don't need this anymore here?

Reply:
Maybe, who knows ?